src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/Application.js

```diff
@@ -97,6 +97,7 @@ export default class Application extends React.PureComponent<Props> {
                             return (
                                 <li
                                     key={index}
+                                    // eslint-disable-next-line react/jsx-no-bind
```

Comment:
I have decided here to go for disabling the rule, since this is part of the navigation, which will be rewritten anyway